test/client/fixtures/redirect-watch-test.js

```diff
@@ -18,31 +17,35 @@ QUnit.testDone(function () {
     settings.get().sessionId = storedSessionId;
 });
 
-function changeLocation (locationChangeScript) {
+function redirectIframe (redirectScript, stayIframe, iframeName) {
     return new Promise(function (resolve, reject) {
-        var iframe   = document.createElement('iframe');
-        var resolved = false;
+        var iframe = document.createElement('iframe');
 
-        iframe.id  = 'test' + Date.now();
-        iframe.src = location.protocol + '//' + location.host + '/unchangeableUrlSession!i/' + iframeLocation;
+        iframe.id   = 'test' + Date.now();
```

Comment:
`Date.now` is a good variant.
But we have a lot of variants of iframe id naming:
iterative - `test1`, `test22`
symbols - `test_unique_id_lsfgj`,
with date.now - `test` + `Date.now`.

I think we should standardize it.